### PR TITLE
Add changes in IoTS-1.0.0-M2 to M3

### PR DESCRIPTION
**Pull request by charithag**
Merge changes

### CODE REVIEW

File: components/device-mgt-iot-digitaldisplay/org.wso2.carbon.device.mgt.iot.digitaldisplay.controller.api/src/main/java/org/wso2/carbon/device/mgt/iot/digitaldisplay/controller/api/util/DigitalDisplayMQTTConnector.java [hunk 1]
```diff
@@ -72,27 +81,55 @@ public void processIncomingMessage(MqttMessage message, String... messageParams)
...
-            String requestMessage = messageData[1];
-            String result = messageData[2];
+        String token = messageData[0];
+        if (messageData.length == 2) {
+            String responseMessage = messageData[1];
+            DigitalDisplayWebSocketServerEndPoint.sendMessage(token, new StringBuilder(responseMessage));
+        } else if (messageData.length == 3) {
+            String response = messageData[2];
+            JSONObject schreenshot = new JSONObject(response);
+            String pic_id = schreenshot.getString("pic_id");
```
Comment (kasundananjaya) [inline components/device-mgt-iot-digitaldisplay/org.wso2.carbon.device.mgt.iot.digitaldisplay.controller.api/src/main/java/org/wso2/carbon/device/mgt/iot/digitaldisplay/controller/api/util/DigitalDisplayMQTTConnector.java]: Follow proper variable naming conventions!

File: components/device-mgt-iot-digitaldisplay/org.wso2.carbon.device.mgt.iot.digitaldisplay.controller.api/src/main/java/org/wso2/carbon/device/mgt/iot/digitaldisplay/controller/api/model/ScreenShotModel.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package org.wso2.carbon.device.mgt.iot.digitaldisplay.controller.api.model;
+
+public class ScreenShotModel {
+
+    private String[] scrrenShotData;
```
Comment (kasundananjaya) [inline components/device-mgt-iot-digitaldisplay/org.wso2.carbon.device.mgt.iot.digitaldisplay.controller.api/src/main/java/org/wso2/carbon/device/mgt/iot/digitaldisplay/controller/api/model/ScreenShotModel.java]: Fix spelling.

File: components/device-mgt-iot-digitaldisplay/org.wso2.carbon.device.mgt.iot.digitaldisplay.controller.api/src/main/java/org/wso2/carbon/device/mgt/iot/digitaldisplay/controller/api/util/DigitalDisplayMQTTConnector.java (hunk 1 above)
Comment (charithag) [inline components/device-mgt-iot-digitaldisplay/org.wso2.carbon.device.mgt.iot.digitaldisplay.controller.api/src/main/java/org/wso2/carbon/device/mgt/iot/digitaldisplay/controller/api/util/DigitalDisplayMQTTConnector.java]: Fixed in commit https://github.com/wso2/carbon-device-mgt-plugins/commit/f90719be70c718497c4c9f95f7d846890216a49c

File: components/device-mgt-iot-digitaldisplay/org.wso2.carbon.device.mgt.iot.digitaldisplay.controller.api/src/main/java/org/wso2/carbon/device/mgt/iot/digitaldisplay/controller/api/model/ScreenShotModel.java (hunk 2 above)
Comment (charithag) [inline components/device-mgt-iot-digitaldisplay/org.wso2.carbon.device.mgt.iot.digitaldisplay.controller.api/src/main/java/org/wso2/carbon/device/mgt/iot/digitaldisplay/controller/api/model/ScreenShotModel.java]: Fixed in commit https://github.com/wso2/carbon-device-mgt-plugins/commit/f90719be70c718497c4c9f95f7d846890216a49c

**Pull request merged**